svg/extensibility/interfaces/foreignObject-graphics.svg

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg xmlns="http://www.w3.org/2000/svg"
+     xmlns:h="http://www.w3.org/1999/xhtml">
+  <metadata>
+    <link xmlns="http://www.w3.org/1999/xhtml" rel="help" href="https://www.w3.org/TR/2013/WD-SVG2-20130618/extend.html#ForeignObjectElement"/>
```

Comment:
Do we really want to link to such an old version of the spec? Aside from that  LGTM. Fix if you want or I'll merge in a minute.